.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 assignees: ""
 ---
 
-_Please note as part of [our Q2 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/6161) we paused adding new wallets until we establish new, expanded criteria._
+Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/). Only continue with the issue if your wallet meets the criteria listed there.
 
-Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/).
+If the wallet does meet our list policy, please complete the following information which we need to accurately list the wallet.
 
-Only continue with the issue if your wallet meets the criteria listed there.
+### Project info
 
-If it does complete the following information which we need to accurately list the wallet.
+**Wallet name**
 
-**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team or some other method.**
+  <!-- Please provide the official name of the wallet -->
+
+**Wallet description**
+
+<!-- Please provide a short 1-2 sentence description of the wallet. Avoid unsubstantiated marketing claims like “the best Ethereum wallet”. -->
+
+**Wallet logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Background color for brand logo**
+
+<!-- Please provide a hex code for the brand color. This will be added to the background of the wallet card. -->
+
+**URL to the project**
+
+<!-- Please provide a URL (e.g. to the website of the wallet) -->
+
+**URL to the documentation**
+
+<!-- Please provide a URL to the documentation -->
+
+**Is the wallet security tested? Please explain security measures i.e. security audit, internal security team, or some other method.**
 
 <!-- Please explain any security measures you have taken to ensure your wallet is secure -->
 
 **When did your wallet go live to users?**
 
-<!-- Please provide an exact or approximate date when your wallet was usable by the public -->
+<!-- Please provide a date when your wallet was usable by the public. Please provide some user metrics for how many users are using this wallet. -->
 
 **Does your wallet have an active development team?**
```

Comment:
**Open question for discussion:** Should an active development team be noted as an absolute requirement?
Also, what defines _active_? 

My take is this should be a requirement to keep users safe. Ethereum is still relatively young and has many upgrades being rolled out, and listed wallets should have developers maintaining the codebase and tending to bug reports. 

Not sure what the cutoff should be though for an "inactive" project... Perhaps either an official announcement, or no code commits for > 6 mos?

Reply:
I do think an active development team is a requirement. Just last year there was the EIP-1559 upgrade that required wallets to upgrade as a single example where there are changes that wallets would need to support. If anything this is almost the main requirement out of all of this.